p1-staking-pool-dyn/src/lib.rs

```diff
@@ -242,14 +235,8 @@ impl Contract {
 
     /// mint cheddar rewards for the user, maybe closes the account
     /// NOTE: the destination account must be registered on CHEDDAR first!
-    fn mint_cheddar_promise_maybe_close_account(
-        &mut self,
-        a: &AccountId,
-        amount: U128,
-        close: bool,
-    ) -> Promise {
-        // AUDIT: TODO: Assert or check that the amount is positive.
-
+    fn mint_cheddar_promise_maybe_close_account(&mut self, a: &AccountId, amount: U128) -> Promise {
```

Comment:
shall we rename this function and remove `close_account` suffix?

Reply:
ok. done.